src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --colorForLinks: #00acdc;
+}
+
+.menu {
+  padding: 0 50px;
+}
+
+.menu__content {
+  max-width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__box {
+  margin-right: 20px;
+}
+
+.nav__box:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  position: relative;
+  font-family: "Roboto", sans-serif;
```

Comment:
add `font-family` style to `body` or `html` selector. then you can use `font-family: inherit`
do not forget to add more fallback fonts
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/5d887d79-d0ef-4385-8497-dda9db855fe0)